addons/base_automation/tests/test_automation.py

```diff
@@ -22,7 +22,22 @@ def test_01_on_create(self):
                 "evaluation_type": "equation",
                 "value": "False",
             })],
-        })
+        },
+        {
+            "name": "Bilbo time senstive reminder",
+            "trigger": "on_time",
+            "model_id": self.env.ref("hr_contract.model_hr_contract").id,
+            "type": "ir.actions.server",
+            "trigger_field_ids": [],
+            "trg_date_range":-7,
+            "trg_date_range_type":"day",
+            "trg_date_id": self.env.ref("hr_contract.field_hr_contract__date_end").id,
+            "fields_lines": [],
+        }])
+
+        # verify the "Base Action Rule: check and execute" frequency is set correctly with negative numbers.
+        cron = self.env["ir.cron"].search([("cron_name","=","Base Action Rule: check and execute")])
+        self.assertEqual(cron.interval_number,240)
```

Comment:
please also assert the interval_type, it could be 240 months we wouldn't know

Reply:
Yeah you're right, i pushed the fix now